AWS/legos/aws_find_older_generation_ec2_instances/README.md

```diff
@@ -0,0 +1,23 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h1>Find Older Generation EC2 Instances</h1>
+
+## Description
+This action finds older generation EC2 instances, which are ['t2', 'm1', 'm2', 'm3', 'm4', 'c1', c3] type.

```

Comment:
Isnt the previous generation list like [this](https://aws.amazon.com/ec2/previous-generation/)